django-verdant/rca/templates/404.html

```diff
@@ -1,11 +1,21 @@
 {% extends "rca/base.html" %}
 
-{% block title %}Page not found{% endblock %}
+{% block title %}Something's not right{% endblock %}
 
 {% block content %}
-    <h1 class="h1">Page not found</h1>
+    <h1 class="h1">Something's not right</h1>
 
-    <h2>Sorry, this page could not be found.</h2>
+    <div class="body-text">
+        <p>The page you’re looking for may have moved or no longer exists. You can try:</p>
+
+        <ul>
+            <li>using our search feature to find specific content</li>
```

Comment:
**Suggestion- non blocking**

I can see why the search page isn't linked here, because https://www.rca.ac.uk/search/ isn't great with no params.
Might be worth populating the search link with the slug/title of the 404: `https://www.rca.ac.uk/search/?q="[title]"`

Although this might bleed to much from the ticket AC and cause run over

Reply:
The links themselves were provided as content from Matthew, so I am in favour of leaving them as is (and waiting for this site to die)